src/DocumentFormat.OpenXml/GeneratedCode/schemas-microsoft-com_vml.g.cs

```diff
@@ -3184,7 +3195,7 @@ public Group(string outerXml) : base(outerXml)
         /// xmlns:o=urn:schemas-microsoft-com:office:office
         /// </remark>
         [SchemaAttr(27, "tableproperties")]
-        [Index(31)]
+        [Index(32)]
```

Comment:
Some of the last magic numbers in this thing. I hope to get them cleaned up in the next couple weeks once my workload lessens a bit.

Reply:
Did you need something changed here? Is the index number changing causing a problem?

Reply:
@twsouthwick or can we proceed with the merge?

Reply:
Oh - that was just a comment that I've been trying to clear up these magic numbers. no action needed, just pointing out where these are for future information :)

Reply:
got it. yeah, I noticed them in the last two changes. I was worried at first that they may cause conflicts or issues but apparently they don't. We should add a workitem in the backend.